h2-engine/2.1.210.wso2v1/pom.xml

```diff
@@ -0,0 +1,119 @@
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied.  See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <groupId>org.wso2.orbit.com.h2database</groupId>
+    <artifactId>h2-engine</artifactId>
+    <version>2.1.210.wso2v1</version>
+    <packaging>bundle</packaging>
+    <modelVersion>4.0.0</modelVersion>
+    <name>WSO2 Carbon Orbit - H2 OSGi Console Service</name>
+
+    <distributionManagement>
+        <repository>
+            <id>wso2.releases</id>
+            <name>WSO2 internal Repository</name>
+            <url>https://maven.wso2.org/nexus/content/repositories/releases/</url>
+        </repository>
+    </distributionManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <version>2.1.210</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.osgi</groupId>
+            <artifactId>org.eclipse.osgi</artifactId>
+            <version>3.9.1.v20130814-1242</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>commons-logging</groupId>
+            <artifactId>commons-logging</artifactId>
+            <version>1.2</version>
+            <optional>true</optional>
+        </dependency>
+    </dependencies>
+
+    <properties>
+        <h2.engine.orbit.version>2.1.210.wso2v1</h2.engine.orbit.version>
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>wso2-nexus</id>
+            <name>WSO2 internal Repository</name>
+            <url>https://maven.wso2.org/nexus/content/groups/wso2-public/</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+                <checksumPolicy>fail</checksumPolicy>
+            </releases>
+        </repository>
+    </repositories>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Private-Package>org.h2.*</Private-Package>
+                        <Export-Package>
+                            org.wso2.carbon.h2.osgi.*;version="${h2.engine.orbit.version}",
+                        </Export-Package>
+                        <_exportcontents>org.h2.*;version=2.1.210.wso2v1</_exportcontents>
```

Comment:
We have marked the `org.h2.*` as private package and then we have this _exportcontents, what will the behaviour of this ?

Reply:
<_exportcontents> behaves just like Export-Package and when the <Export-Package> set and <Private-Package> set overlaps `export` takes precedence [1].
[1] https://felix.apache.org/documentation/subprojects/apache-felix-maven-bundle-plugin-bnd.html